### PR TITLE
ceph.spec.in: add missing BuildRequires from SUSE block

### PR DESCRIPTION
**Pull request by dachary**
http://tracker.ceph.com/issues/11901

### CODE REVIEW

**Comment by dachary** [comment]
@osynge I'm kind of hoping you're online because we're in the same timezone ;-)

**Comment by dachary** [comment]
@ktdreyer just in case you're around although you should really be asleep

**Comment by dachary** [comment]
> insert an %endif at 122
done

File: ceph.spec.in [hunk 1]
```diff
@@ -135,7 +137,6 @@ Requires(preun):	chkconfig
 Requires(preun):	initscripts
 BuildRequires:	gperftools-devel
 %endif
-%endif
```
Comment (osynge) [inline ceph.spec.in]: this has been resolved with https://github.com/ceph/ceph/pull/4858/files ?

**Comment by osynge** [comment]
+ to this fix

**Comment by osynge** [comment]
I am happy for this to be merged.

Am currently working on autotools making this obsolete as else statements in rpm are interesting!

**Comment by dachary** [comment]
now building in gitbuilders

**Pull request merged**